File: atcoder/abc126A_changing_a_character.ml
(* vicfred
 * https://atcoder.jp/contests/abc126/tasks/abc126_a
 * implementation
 * *)
let n, k, s = Scanf.scanf " %d %d\n%s" @@ fun a b c -> a, b - 1 , c
let main = print_endline @@ String.mapi (fun i c -> if i = k then Char.lowercase_ascii c else c) s

File: atcoder/abc126A_changing_a_character.nim
# vicfred
# https://atcoder.jp/contests/abc126/tasks/abc126_a
# implementation
import strutils, sequtils

var n, k: int
(n, k) = stdin.readLine.split.map parseInt

var s = stdin.readLine

s[k-1] = s[k-1].toLowerAscii

echo s

